.travis.yml

```diff
@@ -20,6 +20,8 @@ install:
  - composer install --prefer-source
 
 script:
+ - composer test
+ - composer check-code
  - if [[ $COVERAGE == yes ]]; then composer check-code; fi
```

Comment:
Shouldn't this conditional check and run be removed since we run always  `check-code`  ?

Reply:
Good catch!